netci.groovy

```diff
@@ -29,7 +29,7 @@ def project = GithubProject
             case 'Windows_NT':
                 newJob.with{
                     steps{
-                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" && RebuildWithLocalMSBuild.cmd")
+                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" && RebuildWithLocalMSBuild.cmd /p:LocalizedBuild=true")
```

Comment:
Changing the script to take arguments is good, but I think I'd rather see the default be localized (with an opt-out). Not urgent though.